code/game/machinery/kitchen/petrel_maker.dm

```diff
@@ -2,19 +2,20 @@
 /obj/machinery/petrel_maker
 	name = "Greyson Petrel Pump"
```

Comment:
```suggestion
	name = "Greyson Petrol Pump"
```
I want to believe you meant like, Petrol pump, as in, from a gas station. With an o.
Might need refactor to make it consistent